fulp_modules/features/clothing/body_cameras/body_camera.dm

```diff
@@ -1,54 +1,87 @@
-
-/obj/item/clothing/suit/armor/Initialize(mapload)
-	. = ..()
-	AddComponent(/datum/component/bodycamera_holder)
-
-/obj/item/clothing/suit/hooded/wintercoat/security/Initialize(mapload)
-	. = ..()
-	AddComponent(/datum/component/bodycamera_holder)
-
-/obj/item/clothing/suit/security/Initialize(mapload)
-	. = ..()
-	AddComponent(/datum/component/bodycamera_holder)
-
 /**
- * The bodycamera
+ * Bodycamera Upgrade
  *
- * This is the item that gets installed into items that have the bodycamera_holder element
+ * An item that can be inserted into any piece of clothing that goes on the Exosuit slot.
+ * Allows you to turn it on/off with an ID card to add the suit & the wearer's name into the security camera system.
  */
 /obj/item/bodycam_upgrade
 	name = "\improper body camera"
+	desc = "An armor vest upgrade, there's an instructions tag if you look a little closer..."
 	icon = 'fulp_modules/features/clothing/body_cameras/bodycamera.dmi'
 	icon_state = "bodycamera"
-	desc = "An armor vest upgrade, there's an instructions tag if you look a little closer..."
-	///The camera itself.
-	var/obj/machinery/camera/builtin_bodycamera
+	///The camera itself, made when we need it and deleted on Destroy. Installed into the clothing item directly.
+	var/obj/machinery/camera/bodycamera/builtin_bodycamera
+	/**
+	 * Sprites by: @Partheo from Yogstation, colors very, very slightly edited.
+	 * A static overlay put onto any clothing item that has the camera installed.
+	 */
+	var/static/mutable_appearance/equipped_overlay = mutable_appearance('fulp_modules/features/clothing/body_cameras/bodycamera_overlay.dmi', "bodycamera")
 
 /obj/item/bodycam_upgrade/examine_more(mob/user)
 	. = ..()
-	. += list(span_notice("Use [src] on any valid vest to instantaneously install"))
-	. += list(span_notice("Use a [span_bold("screwdriver")] to remove it."))
-	. += list(span_notice("Once installed, the camera will activate/deactivate whenever its associated vest is worn/removed."))
-	. += list(span_notice("You can also use your ID card on the vest to toggle its camera manually."))
+	. += list(span_notice("You can use [name] on any vest to install it, or a [EXAMINE_HINT("screwdriver")] on the vest to uninstall one."))
```

Comment:
```suggestion
	. += list(span_notice("You can use [name] on any suit to install it."))
```
"Suit" might be accurate than "vest". Also the component itself already adds information on how to remove the camera.

Reply:
Suit can also mean jumpsuit

Reply:
True. "Outer layer clothing" it is then.